### PR TITLE
policy sync endpoints for sam automation tests

### PR DESCRIPTION
**Pull request by marctalbott**
**PR checklist**
- [x] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [x] Bump the version in `project/Settings.scala` `createVersion()`
- [x] Update `CHANGELOG.md` for those libraries
- [x] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [x] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [x] Get two thumbsworth of PR review
- [x] Verify all tests go green (CI _and_ coverage tests)
- [x] Squash commits and **merge to develop**
- [x] Delete branch after merge

### CODE REVIEW

File: model/src/main/scala/org/broadinstitute/dsde/workbench/model/SamModel.scala [hunk 1]
```diff
@@ -9,7 +9,11 @@ object SamJsonSupport {
...
 
   implicit val AccessPolicyResponseEntryFormat = jsonFormat3(AccessPolicyResponseEntry.apply)
+
+  implicit val CreateResourceRequestFormat = jsonFormat3(CreateResourceRequest.apply)
 }
 
-final case class AccessPolicyMembership(memberEmails: Set[WorkbenchEmail], actions: Set[String], roles: Set[String])
-final case class AccessPolicyResponseEntry(policyName: String, policy: AccessPolicyMembership, email: WorkbenchEmail)
\ No newline at end of file
+final case class AccessPolicyMembership(memberEmails: Set[String], actions: Set[String], roles: Set[String])
```
Comment (helgridly) [inline model/src/main/scala/org/broadinstitute/dsde/workbench/model/SamModel.scala]: aww really? this makes it a breaking change

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Sam.scala [hunk 2]
```diff
@@ -108,12 +108,12 @@ object Sam extends Sam {
       deleteRequest(url + s"api/groups/v1/$groupName")
     }
 
-    def addUserToPolicy(groupName: String, policyName: String, memberEmail: String)(implicit token: AuthToken): Unit = {
+    def addUserToManagedGroupPolicy(groupName: String, policyName: String, memberEmail: String)(implicit token: AuthToken): Unit = {
```
Comment (helgridly) [inline serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Sam.scala]: if you want to rename these functions, go ahead, but leave the old ones there + mark them as deprecated + make them call the new ones

**Review by helgridly — CHANGES_REQUESTED** [review]
i think there are a couple minor things you can do to make this an entirely nonbreaking change

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Sam.scala (hunk 2 above)
Comment (Qi77Qi) [reply serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Sam.scala]: if we know this function is not used by any code other than sam, I'd rather just rename them to reduce maintenance overhead.

**Review by dvoet — APPROVED** [review]

File: model/CHANGELOG.md [hunk 3]
```diff
@@ -2,9 +2,17 @@
 
 This file documents changes to the `workbench-model` library, including notes on how to upgrade to new versions.
 
+## 0.13
+
+SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-model" % "0.13-TRAVIS-REPLACE-ME"`
+
+### Changed
+
+- Remove duplicated `SamModel`
```
Comment (Qi77Qi) [inline model/CHANGELOG.md:11]: I think you'll need to bump other modules that depend on `model`, which is all modules I think.

Comment (helgridly) [reply model/CHANGELOG.md:11]: yes (sadly)

Comment (marctalbott) [reply model/CHANGELOG.md:11]: Just to be clear, you mean that all the other libs need minor version bumps as well, is that correct?

Comment (helgridly) [reply model/CHANGELOG.md:11]: yes

**Review by helgridly — APPROVED** [review]

**Pull request merged**